### PR TITLE
Fix white non bfb

### LINKED ISSUE #485: Main branch has a test failure on cdash
This [cdash error](https://my.cdash.org/viewTest.php?onlyfailed&buildid=1862147) shows that the main branch has a broken test. The 7-27 cdash doesn't have the error, so it started sometime in the last week.

### PR DESCRIPTION
**Pull request by jgfouca**
Needed to use a bfb tanh in MurphyKoop_svp

Fixes #485

### CODE REVIEW

File: components/cam/src/physics/cam/micro_p3.F90 [hunk 1]
```diff
@@ -1558,7 +1558,7 @@ real(rtype) function MurphyKoop_svp(t, i_type)
        ! (good for 123 < T < 332 K)
        !For some reason, we cannot add line breaks if we use "bfb_exp", storing experssion in "tmp"
        tmp = lq(1) - (lq(2) / t) - (lq(3) * logt) + (lq(4) * t) + &
-            (tanh(lq(5) * (t - lq(6))) * (lq(7) - (lq(8) / t) - &
+            (bfb_tanh(lq(5) * (t - lq(6))) * (lq(7) - (lq(8) / t) - &
```
Comment (jgfouca) [inline components/cam/src/physics/cam/micro_p3.F90:1561]: This is the actual fix.

File: components/cam/src/physics/cam/micro_p3.F90 [hunk 2]
```diff
@@ -3622,9 +3622,9 @@ subroutine rain_sedimentation(kts,kte,ktop,kbot,kdir,   &
                     mu_r(k), lamr(k), V_qr(k), V_nr(k))
 
                !in compute_rain_fall_velocity, get_rain_dsd2 keeps the drop-size
-               !distribution within reasonable bounds by modifying nr_incld. 
+               !distribution within reasonable bounds by modifying nr_incld.
                !The next line maintains consistency between nr_incld and nr.
-               nr = nr_incld*rcldm 
+               nr(k) = nr_incld(k)*rcldm(k)
```
Comment (jgfouca) [inline components/cam/src/physics/cam/micro_p3.F90:3627]: Fix for problem Andrew found.

**Comment by singhbalwinder** [comment]
> Great! Why do we still have two versions of these functions, though? Thought the redundancies were removed at some point.

PR #469 removes these redundancies but it is yet to be merged to master.

**Comment by jgfouca** [comment]
@ambrad I was also disappointed about the duplication of the bfb math stuff. I will look into removing the duplication when I launch into SHOC.

**Comment by jgfouca** [comment]
The failures in the autotester are expected since this PR is not BFB. I've confirmed that both p3_run_and_cmp tests are working on white if the baselines are made from this PR HEAD.

**Pull request merged**